stdlib/public/core/CMakeLists.txt

```diff
@@ -238,29 +237,6 @@ set(swift_core_link_flags "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}")
 set(swift_core_framework_depends)
 set(swift_core_private_link_libraries)
 set(swift_stdlib_compile_flags "${SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS}")
-if(SWIFT_PRIMARY_VARIANT_SDK IN_LIST SWIFT_DARWIN_PLATFORMS)
-  list(APPEND swift_core_link_flags "-all_load")
```

Comment:
(from outside) Was ICU the only thing using `-all_load`? Or is this something that can be removed separately? Or is it still needed?

Reply:
Yes, ICU is the only think that the standard library links against (outside of system libraries like ... libSystem) explicitly.  The rest of the dependencies that need to be linked against will be from the clang triggered auto-link.  `INCORPORATE_OBJECT_LIBRARIES` does properly use the `TARGET_OBJECTS` generator to unpack the archive and link them so no unintentional DCE should occur.

Reply:
It was swiftRuntime.a itself I was worried about, but if that's going through `INCORPORATE_OBJECT_LIBRARIES` then everything should be okay.

Reply:
Right, that is handled via the incorporate object libraries and so we don't need to worry about the potential DCE.